ApplySyntax.py

```diff
@@ -559,6 +561,34 @@ def function_matches(self, rule):
             else:
                 return False
 
+    def filesize_matches(self, rule):
+        """Perform file_size match."""
+
+        size_rule = rule.get("file_size")
+        file_size = os.path.getsize(self.file_name)
+
+        rule_len = len(size_rule)
+        if (rule_len <= 1):
+            return False
```

Comment:
makes sense - should I update this, or do you want to wait to make it a global feature?